classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :student
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
```

Comment:
- Hey, great one here, so we also need to add the relationship between the student and the classroom here, so let's add this missing part below `student.classroom = self`:

```ruby

  def add_student(student)
    @students.push(student)
    student.classroom = self
  end
```